src/Task02_PokeAPI/pokeApi.js

```diff
@@ -0,0 +1,172 @@
+export async function fetchPokemonData (pokemonName) {
+  const graphqlQuery = {
+    query: `
+      query getPokemonDetails($name: String!) {
```

Comment:
you can also keep graphql queries in separate file as constants. it will be much easier to read your code.